test/integration/fixtures/server12/mocks/dummyservice.js

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (c) 2015-2016 PointSource, LLC.
+ * MIT Licensed
+ */
+exports.init = function(logger) {
+    logger.info('Dummy Service Mock initialized');
+    process.exit(0); // exit test here
```

Comment:
I think it would be a better test for the dummy service not to exit, but to provide a function that can be called from the test after server startup has completed